tests/vmi_configuration_test.go

```diff
@@ -1136,6 +1136,7 @@ var _ = Describe("Configurations", func() {
 				_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(cpuVmi)
 				Expect(err).ToNot(HaveOccurred())
 				node := tests.WaitForSuccessfulVMIStart(cpuVmi)
+				// what should i do here it will always pass for us (false positve)
```

Comment:
I believe you will need to save nodes where runs CPU manager(you can check CPU manager label) and check according to it